test/jdk/javax/swing/JEditorPane/JEditorPaneFontFallback.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2022 JetBrains s.r.o.
```

Comment:
It seems that different files use different style of "Copyright 2022 JetBrains s.r.o", is the current correct?

Reply:
To my knowledge, it's correct. It's also used in other JetBrains source code currently.